e107_admin/admin.php

```diff
@@ -245,7 +246,8 @@ private function checkTimezone()
 
 		if(e107::getDate()->isValidTimezone($timezone) == false)
 		{
-			$mes->addWarning("Your timezone setting (".$timezone.") is invalid. It has been reset to UTC. To Modify, please go to Admin -> Preferences -> Date Display Options.", 'default', true);
+			$message = e107::getParser()->lanVars(ADLAN_188, array('x'=>$timezone));
```

Comment:
As mentioned above: `$message = e107::getParser()->lanVars(ADLAN_188, $timezone);` should do the same thing.